docs/link-flow.md

```diff
@@ -46,3 +46,10 @@ subscriptions.saveSubscription(() => requestPromise));
 
 The dialog will prompt the user to opt-in to save the subscription. If the user agrees, the provided callback will be called to resolve the token/authCode. The resulting promise will be resolved once the subscription has been saved. 
 
+# SwG Login Prompt flow
```

Comment:
We should either remove this section for now, or add "(not launched)" suffix.

Reply:
done